Sample Data/Custom/GCP_DNS_CL.json

```diff
@@ -0,0 +1,707 @@
+[
+    {
+        "TenantId": "00000000-0000-0000-0000-000000000000",
+        "SourceSystem": "RestAPI",
+        "MG": "",
+        "ManagementGroupName": "",
+        "TimeGenerated [UTC]": "6/17/2021, 9:20:01.775 AM",
+        "Computer": "",
+        "RawData": "",
+        "resource_labels_zone_name_s": "",
+        "payload__type_s": "",
+        "payload_authenticationInfo_principalEmail_s": "",
+        "payload_requestMetadata_requestAttributes_time_t [UTC]": "",
+        "payload_serviceName_s": "",
+        "payload_methodName_s": "",
+        "payload_authorizationInfo_s": "",
+        "payload_resourceName_s": "",
+        "payload_request__type_s": "",
+        "payload_request_project_s": "",
+        "resource_labels_policy_name_s": "",
+        "payload_request_managedZone_s": "",
+        "log_name_s": "projects/sometestproject-1111111/logs/dns.googleapis.com%2Fdns_queries",
+        "insert_id_s": "785cqudte9v",
+        "severity_s": "INFO",
+        "timestamp_t [UTC]": "6/17/2021, 9:15:33.277 AM",
```

Comment:
"timestamp_t [UTC]" fields value ingesting in LAW as string , due to which parser is failing, can you please provide both fields values as per following format in all places.
"timestamp_t [UTC]":"2021-06-16T02:27:00.00000Z",
Also please remove the [UTC] from the column name.

Thanks,
Aditi Maheshwari

Reply:
updated